Adds 10 more pokemon

Lindayezou opened the pull request:
Adds 3 greens, 3 reds and 4 blues

 Changes to be committed:
    modified:   server/data/pokemonData.js

Commit:
Adds 10 more pokemon

Adds 3 greens, 3 reds and 4 blues

 Changes to be committed:
	modified:   server/data/pokemonData.js

chououtside commented on server/data/pokemonData.js:
```diff
@@ -112,5 +112,196 @@ module.exports = [
+      attackStrength: 5
+    }
   }
+  ==========
```
Get rid of this line **test before submitting PR**

Commit:
Takes out line

Deletes line that is breaking server

 Changes to be committed:
	modified:   server/data/pokemonData.js
